yt_dlp/extractor/netverse.py

```diff
@@ -0,0 +1,167 @@
+import functools
+
+from .common import InfoExtractor
+from .dailymotion import DailymotionIE
+from ..utils import (
+    ExtractorError,
+    GeoRestrictedError,
+    OnDemandPagedList,
+    smuggle_url,
+    traverse_obj,
+)
+
+
+class NetverseBaseIE(InfoExtractor):
+    _ENDPOINTS = {
+        'watch': 'watchvideo',
+        'video': 'watchvideo',
+        'webseries': 'webseries',
+    }
+
+    def _call_api(self, url, query={}):
+        display_id, sites_type = self._match_valid_url(url).group('display_id', 'type')
+
+        json_data = self._download_json(
+            f'https://api.netverse.id/medias/api/v2/{self._ENDPOINTS[sites_type]}/{display_id}',
+            display_id, query=query)
+
+        # adapted from dailymotion.py
+        error = json_data.get('error')
+        if error:
+            # See https://developer.dailymotion.com/api#access-error
+            if error.get('code') == "DM007":
+                raise GeoRestrictedError(error.get("title") or error.get("raw_message"))
+            raise ExtractorError(f'{self.IE_NAME} said: {error.get("title") or error.get("raw_message")}')
```

Comment:
Show example of when this code is reached

Reply:
Here is the yt-dlp log:
```
(python_proj) ~/.../python_proj/yt-dlp $ python -m yt_dlp -sv 'https://www.netverse.id/video/pg067482-hellojadoo-season1'
[debug] Command-line config: ['-sv', 'https://www.netverse.id/video/pg067482-hellojadoo-season1']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.05.18 [b14d52355]
[debug] Lazy loading extractors is disabled
[debug] Plugins: ['SamplePluginIE', 'SamplePluginPP']
[debug] Python version 3.10.4 (CPython 32bit) - Linux-3.18.14-14381225-QB20734933-armv8l-with-libc
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 5.0.1 (setts), ffprobe 5.0.1
[debug] Optional libraries: Cryptodome-3.14.1, brotli-1.0.9, certifi-2022.05.18.1, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] [Netverse] Extracting URL: https://www.netverse.id/video/pg067482-hellojadoo-season1
[Netverse] pg067482-hellojadoo-season1: Downloading JSON metadata
[debug] [dailymotion] Extracting URL: https://www.dailymotion.com/video/x887jzz#__youtubedl_smuggle=%7B%22query%22%3A+%7B%22embedder%22%3A+%22https%3A%2F%2Fwww.netverse.id%22%7D%7D
[dailymotion] Downloading Access Token
[dailymotion] x887jzz: Downloading media JSON metadata
[dailymotion] x887jzz: Downloading metadata JSON
ERROR: [dailymotion] x887jzz#: Video mit geografischer Einschränkung.
This video is available in Indonesia.
You might want to use a VPN or a proxy server (with --proxy) to workaround.
Traceback (most recent call last):
  File "/data/data/com.termux/files/home/python_proj/python_proj/yt-dlp/yt_dlp/extractor/common.py", line 642, in extract
    ie_result = self._real_extract(url)
  File "/data/data/com.termux/files/home/python_proj/python_proj/yt-dlp/yt_dlp/extractor/dailymotion.py", line 266, in _real_extract
    self.raise_geo_restricted(msg=title, countries=allowed_countries)
  File "/data/data/com.termux/files/home/python_proj/python_proj/yt-dlp/yt_dlp/extractor/common.py", line 1149, in raise_geo_restricted
    raise GeoRestrictedError(msg, countries=countries)
yt_dlp.utils.GeoRestrictedError: Video mit geografischer Einschränkung.
Traceback (most recent call last):
  File "/data/data/com.termux/files/home/python_proj/python_proj/yt-dlp/yt_dlp/extractor/common.py", line 642, in extract
    ie_result = self._real_extract(url)
  File "/data/data/com.termux/files/home/python_proj/python_proj/yt-dlp/yt_dlp/extractor/dailymotion.py", line 266, in _real_extract
    self.raise_geo_restricted(msg=title, countries=allowed_countries)
  File "/data/data/com.termux/files/home/python_proj/python_proj/yt-dlp/yt_dlp/extractor/common.py", line 1149, in raise_geo_restricted
    raise GeoRestrictedError(msg, countries=countries)
yt_dlp.utils.GeoRestrictedError: Video mit geografischer Einschränkung.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/data/data/com.termux/files/home/python_proj/python_proj/yt-dlp/yt_dlp/YoutubeDL.py", line 1399, in wrapper
    return func(self, *args, **kwargs)
  File "/data/data/com.termux/files/home/python_proj/python_proj/yt-dlp/yt_dlp/YoutubeDL.py", line 1469, in __extract_info
    ie_result = ie.extract(url)
  File "/data/data/com.termux/files/home/python_proj/python_proj/yt-dlp/yt_dlp/extractor/common.py", line 668, in extract
    raise type(e)(e.orig_msg, **kwargs)
yt_dlp.utils.GeoRestrictedError: [dailymotion] x887jzz#: Video mit geografischer Einschränkung.
```

And this is the json response:
```json
{
  "url": "https://www.dailymotion.com/video/x887jzz",
  "error": {
    "title": "Video mit geografischer Einschränkung.",
    "message": "Suchen Sie nach „<a target=\"_top\" href=\"/search/Namaku%20Choi%20Jadoo\" class=\"highlight\">Namaku Choi Jadoo</a>“ oder entdecken Sie mehr Videos auf <a target=\"_top\" href=\"/\" class=\"highlight\">Dailymotion</a>.",
    "raw_message": "Suchen Sie nach „Namaku Choi Jadoo“ oder entdecken Sie mehr Videos auf Dailymotion.",
    "code": "DM007",
    "status_code": 403,
    "refresh": 0
  },
  "duration": 780,
  "id": "x887jzz",
  "title": "Namaku Choi Jadoo",
  "view_id": "1g502u08f3l840mlvdk",
  "private": false,
  "data_center": "dc3",
  "access_id": "x887jzz",
  "is_password_protected": false,
  "advertising": {
    "ad_url": "https://dmxleo.dailymotion.com/cdn/manifest/video/x887jzz.m3u8?auth=1654812031-2688-rdurulqz-aa4ce04dc057ac2e002e5d52992490d686G_EASIqZsHF_pdk2IRQq2Wuq0vx5_CxGhliIWw2RJW6efDT6jFqfbmrz9EnUgs0kfpY2Fpp6rhIkoyuzIfX3B-e87mTH9AVzrakgvpgDZeA1KyMFMh185JBmWYriJem1z0fef6wLNx8MgHgGEBfH9LNgytTF4r2zLWP-o1-qxrQavQkZDgguG_aOBaL4fWjcM9YFyasXph3tpVW-NoryYKh_cc-ngve2Es-o8jYWO1V-kQxHc9BiFYoy0oxUvdXPH1sAozKvG79iuj06u148vygOoIH_kCs_sRbkJy1t_3HC4QYIXXYBs9Miir-l1jxyAiADQBeMd9fAzmXigykS2drzKQ4m4vQO3YXhVMMV-OpkVdrq9N3w1BCUMSJbKFA7qdRt1FVevRqWTCEP2b-C8rEBxhM3nBWf9yS1z9fitacJ0_ocsq1iQosq_kzNpUcVdejggCswUmZF9131AmI1j0UGmvwn3aj8o_DHPNcNCR56B92rZegu9m4LSVTSPAcuu_G7le8_UnsWlqSVgkx-CE52dedTpcsR6IkFUmyErukrOFrf6sEd1gV9Y7o7B5PT189_Knlk3LZ9hJ8OqQ2ZBP9QfjNjJaIzgWEMjZ-mM8IzpSTtUMLncxOXJmyaVOgKfcVijRnjHFLcRh9C2UnOmT7ROj_lET1ZD948ypYkN30kthUnbFIMCLHngKcOYEJgnOcF6-8fkx_QTlDSHl1fcvMVRrOdRMhMNrzgFLwDVvDWmpK7YOKlbW35-aZfBtfs9df6bC3hpXAWeDSY9ZouEBVwA4WCD5_vjBbQFonVcu4pC8FViXWH8X9Q8w5v14c0MN839sm_QMYjUOKTSHkRiil2MsbNJIoI4TfFOczJvQ61L_m0G2P1PuWRDjhexfDVG2cPIOv48bULYcxq12ZAYvM6BisONcNsvOQDcqVuKMdevl_mlKd4VrVgEzbolgcQcDmHllLaQFTsxf4Yn2P25yErpYDwiMZ2Xlegq9QaZCDwwt7LADszoCA8XAFnqVybuME5Gg7Db2vc66mTVzMv2zI8JFLeSxzokaJ-l1Fzy03k3Ghgap1laaV4vvksbYyF_D7WZcXVY8rVyuESZlZG9mjEnv6XHCVqpYQvNORg6kf43H8Dxmkt3fBxMmJK0AcA2mkFYbPed9wna-uRqJDxatSB-_npXBavPnH1v5ATYAtnTIygSIzNv9BZuw7_6b6lSFjYgUydSE4Z3mhsBxgn_1Wg-tIeHLylryn1GWPX8Vae2KrUMRFA8A-lF3LUJW9LqNPO3C2zPLAw0nhH-Ujcn0765na2gE4vvbQDgWXBX3jPGnwQRd_eZe-9SHfK_tlTBbDYwfMi7fWjS2_qPE-Svs2VF9n9G-vTjfBQu4G93b5xGV0NlbMDAo0CySNY9re32feGPJMPsKpQJv7ZicbAJwUAknG8svNTpU0EIDfITmMYBLtuvm7_mcqAvXLT1IYyuw_N6wcvXFs8b5wBdVcO7NejNhKH5-AD9BRaigUohX_GbUVXAAPPgpUk6FiYDUhQI6Mc-LXbAfkXAgfL4PgyahdQBpAnCsQ5OYD1WSgxO5y09LmExc27OqFzBHZZrYtRxksVeYiBywrHvz_s-_EndCXvHxuvK-mh9vqvu1gtCvIksy073o5MFVCE-6hmpSAtPZB0wgsl0RpZdh7HJsbWL6f0Cl-W5QfOQ9_Y8sLTRRJgAADWcO3f8dpppCaeTpunfwmD0jxf6URYOKo3hpOITDzjPD6Pub2xIyF576OeJJTWTuuDIWmMytt8_14H_qKgPrJemyLfNZZTeyuaHhHI7wPCaInYQFU5Q7iC_LU5PDiD3O5cFi52i1c4iuGS_mdgsgjy6KUa8O0uBJbr9WY7tXGoFicusrFmxgrbUJfn2Zy_mbX0zgxLpXSnuTxA6napNOYWDowShvdfPuYOcYb3aEr66vBexJAwTsXP1Qnf8aQxodSniIOSOD2puxBoXYQUjJCG3eZyUMkFqz4b93xwOrunAP_CFugZGFPpIafxnVLFIS4-PmirGsHBsgmDvZk51NHrCZYetBTimpQqsccZs7u1gI08mCKSxVNJrah4nb9BZ4de5ww2cZixSnvAJnEPpyPlzOtED79fKVpjUIZedmFVYoGUclogysfATb9vmUwhoLOy-A-a_rkL0qkTfEO1fCZgUGNa-RxQlC2g-FheNzpJ6s6bKnjuE9p2X0ThOilcBHCSmR-KfFXcZIWQyuO2FAQxh2JKB2FZ_A0RWIXTz4E6THdD-zQFHqQMMRnl6pNZ7z1zIiGiuJNDv00X_INEZEhDA9ke_BN-EFBZnp2gS1XlJVhW_35m4Kr8HHIKFUCpCkKedmyo7IqtoEjm0cB3gWJj9dq247A4DXtzhWVJO7UUbIvLQ_8Ml0pDJE8AyK6adoof7gQNeS-I0R11W-oeRW-u8BGaLai5VyRPREW3B8TiwnsSoGqncUBDoy22kaU6lnomYaAzusWCJioaVPO3UkKd6xq2aICxU7P8WlO73oZhce86xXVmkxjLRzXYXMlK1eMJd2qlwBV-6ghMGSCyu8qIraCNhVMfhw0-LUcc-Xa6APwcIm7iKgOp_wC5tev4k2Q4Q3-YJ5aSdlNWBIUw7fXiQUIqFX1SnPOW5SYwFJFBosygk6MQM8wvx8ntnoX2iuzweb2pQYxUWVi90oy8SPXGX-Fd_Ly1-a7DDE_rHC2JRrpOWttOFateRlJHX1R67kTNfiQvg5Yum_bW7YR9f3zJYSi0vS7J6m57KrS_pcqt_8VSXTCC8Ret25oaPmiyfT9LyRGiBd67til5mZUJ7scv8R0-J6vye5WY-zmWz8kGg5JQ73VUVu1z6WB-p32ahRRwjlig-BG4SFZQH7ughRxTzIY8OWcnfsptHq7YHiUVGO9xsjra5DiSEIWu7U5HHbCANmXESYWi3e4xFlccB204TLlj81fW5-8QKdqs2V-cqzY_fj6MRD5IEnqR1Pxw_Ivzab0YwypBejQZBfG",
    "ad_error_url": "https://dmxleo.dailymotion.com/cdn/manifest/video/x887jzz.m3u8?auth=1654812031-2562-vyqzs5nh-3779decf9d1987fb8c86d8a42db56b33zRoJPBknGL_VvZc0nR5sJnv4elbC95XSh8KF4Wxo6P3iVTFYy7KPx_oQmpCNbydHia7m1lM-t3pRlSngstpZFZcsD0e4lp25gTpyKCFD_SyaXu4YpOHSmUAjyaCFXHJVjO66nc6gIh6FgAfZaYyTlwiZ_CcF8UXdNXaxL0c-9KTeJ2Z-_huGYSYTlis0hmUCB4hi9U5HdxWx0fd4XrWbkWLZ-S7wTwkqrf3YVJO4sXtHGnP6eNB23UffOqr0Qy3ZAGZvLcz-VjkRRX06qXN0w425fQpqsfV_k34qxebF-ODyeDEGv_r-6UJRKDxR9Adsh4TQevYCyduuvGSTcKWWem1d3ZgzYxuP2jQMh745Yu4NmA8nesAF0HPFXytshev8dfO4Ib_69hVOudpdSkeyoVqQgSRn20qPPN_HLqAuQFHxGorU1pVzXOpGJ8hx9Tt0otLpEBteApf1mHx0UBf0_vrylCWZPHges9wNj4C5ZWl3-uIf8YaMTts9sykHj99qrLian2tpIpHr4bJrkLlm60SRnLf9KrYgP09QPbmENRi8IuQq8NKaI6YGWMRM1Cyev5K_txnvApbDPPZettTtuSrDWUZo9QYGBwJCjd1ML7Cmo3txW9hgI2o8AVqSVu0lojQY5Plv3p22ETi1w8TwTM30Z3QSQjozxSU6-D5CqdfjzaKxfAXB8jDzuFBb2yACeKGcZ9-5q1jvGqFtKoHp55WPAPLGyG_GUNyMl5rPLj3VOd5OK7ZWXARzyi6ED3E7Ss1Q3DXwloviHTBc1eVY7cQqhOgdUfV-9KPAQ0XnC8ztEKDIP06uVNXn3LwNCUjNP8XDpfEABjFWtUJgp8xLZgYSKooLI6kikaxMGXa4x_9Ne2FoxzaUOVnBfgO_SMNtlC-VBSDuSTeoaDFN3RiZUfdNsCY8pbwJGSEFMMtz1WkUDNwGhDHaJdmfG7z3cOQ50_Q-tIh7T_gfcM0TggXJpiIuAd4ujrMvAwRhJmGG9x1VqRoWCYKndRc420RXp8mg9BCLXwVg7gcIUXEJpueLeqALaVPTe2rLvdQkkbnpso_i2ArvOaZhWPl44HLexINuQxw1ZiyyGlQg6PQthtai0fxFvb4_ksmGdIi-cAMpQjtaQU-PDAMdcSltFZSpWic8WXCr_BjJ9SI-0bkITbtXHUGUMtJKWV1L902ACAfwP7cIyi12XR6T6HdzhivvtXADR1N_XgaufsjvyhZatMpBWA_YUGhOkptTYWY78hYsXtzMYzO-sJBfxYIiF66UAcGFndwyfRfIs8L3sT5EQMFwgT5ir4SjJtdmwQjHHCOWBsGAzGd_mdSQ9FaZSajdHkfKMm6Xmli-AmVKnxEcxV3NjnRRr49oDGB19Gl11tNP3stubB9HS_nTfog4ywDwYzjMvGUuRxzdOu-UWJzolIY5qh3DhUXkSeSQm6cSXtr8V0XntjWG--iWXLddkAqi_NfP9_5rY-X3hrORcX_vACwW-N8Ep4QjbZD2F0LJwPimj_Q8jQKGZA9bXF7OP3XzEVyBz9MXxABNhygEn62h16rZ4F89jLvD0iwc3uGzFbbBhc8CrCiNS091Ra2C5plLMfzAeg-ICTPdL1F2TYqfboiP8LQiOcWvZhe4KfBaKuX4PeXLPdboE564fNIZdI_wOzJ6_brQR_wTtpLl6HHZt49PvqH0BPITM2g0xHrvP--YPm4PHyWcl011pqQHsLpHh_-Vd8fPUTmgqKIV38DHag4b8B72a_BHAV4CDdqgf5nd0bCcJQu6CeMQNyOE64JXLZacazwMRnqjN_NI6q04zs1w7NgKld60mGHRkFCWTUoDWOKEaK8mM1qKNZmcMRmNO2J0ydwtKb2L0LnYaCaOZKMV6px-0bwu5C8ve644s0TU25ePuKqsN_HNFXAEbrMqCnupq_idQvJXV3svPJuWfdUIRGAeBvWC_2olZQcD6kCtcsEw_zotsFp0p2kctnldPs4tBh5iwo2WPaLt88aVjP6wbuRhgt1LRsTXJrbSV2SjG2niX4Bo8ov-BvFUBZFe8WUSwA_oNJFzU-bahunovkqEH6iQLm_ukE6Ps9p6_GTjlwRUuR_ZWaM0PpRftl7mmiSyn--l8roaUTOxpGjxw2pxNlW0hpIhOIsHeACclivJb4SXP0bIA1Fn1q0K8nnHAhrwP__ybCe4QaJZM8gp0k5lqRAdOcfVRGnJUZuv7PSyS9E5_vqfkR2ZpgXlL-6sUPFXHaRUCSqGGTK7OJKPL8Fg7BwXXmRWxWf2AFSx2jFCW24KPgdrTt8y5Ct1eZJPCLmvbaMi2GlF5-iGw2X7E-eGKeuR_hYih3tuSqGgZ2nbh7qY_BymECjvKPKUETFPbxBFzPFl0ROypJZtr2AG5JOl4mgLmBB950rrkSBaBYsHgiMGxSbvHRWR-c4MayaolWdbVnyXpFo6xbhhn8nQ2pQyMhqoBCsgb0uXeAM8KpU-BzBJkNt1f8OYxO9WsdINPTPihfWwQtJHBMo4lCzIaRsJe2lxhYcd82AXlqLpG2ZKde67Ag1dFVa8l1n9U3kAVBc_23lB9YhfGvAJdz8NcFDpKv8G146FvLCKLQ0hKaeARbgVXn3wpWJvrnL4kOAukK2sRIgtsXTDkf_udGkCBUl1BQ8Sbz1nkQvYRenC4PcWeZnGhUlbD8ybr7NSMC3ej4ZBmM3z1drmP-X4lQBTw_hV2eiAN6a1SfFk2cWnu23PeXOOYKDIWE6pqbDR6-JSPUXhZMwIoMxCmNc2vlj2IprM07d07rdAitELu0oDRWC_vDwJ9fbxYwIfUoxZUPBGLNDmwAc1p84qnh6aRh2d9hFXe8_WLHW_RgejfkMtjXitK25dYC8mDkqBZlnyDwUNQqbnmn3lsy8QvBQJpbdWaFwAqdixtPPIqtZLEe-TWslgi8oVwiIjUB8nfCk2SAzfQ-cuEqJYRPuyA00KN9oRj8m-dVr3e3TABOnsuQfzmWf85fUSeEGqz6RhSTU8785rP4m47e7gLJKxtJT-9_MVtPhBp-w&error=[ERRORCODE]",
    "ad_sync_script_url": "https://dmxleo.dailymotion.com/cdn/manifest/video/x887jzz.m3u8?auth=1654812031-2562-4jwft2m6-a8ea8b46173c54248e4d9a160b06b20fbijnSthdb_FNtNZcj1j7MEW8481qMbQZkA8KhXwKVAb2Vij0exDwvfjL1V6-u12QTdt5A7qRwQgN-PuHic59OCLtbOzdCCwUzhe-5pQun6KjwE0StVXGzd995J3cSnagXhiFe2mdq7vxTflb-dpo0bzTNdaQujQ8O2YU_C_6FIxow6rSV7bKoGGkM9MQ4E-gUBYeDVxi-kq-iFCplpO7RH3_Ewmk_s22c9EdbOROlyMzsVREzr7IHLdIdtNpWFsG3E9_6_UnnpBiA0lu1qLAK8_eT3sN4WqDWyXdIq75RZgQUkat2rcaESZ-J3RRyK0Meq5bxAIGbPGCIOlwAZQw1TpAEt6FgCY5UE4YN4_RHY7fr18djJnUjwIGChZaDFdRJ4Gc9vGaiWC3ksUFWnpmaCeheU6IyhYKsWUZoV-EdVo2ZFDUJeOeINF0nMQ17cYfIR-xCv2p8PT_nVO2GM4qlSv_CL94Su5RWs0_EVIcJL2T9-WyDtKy2IKlQVpxjBCY9sVCohAZkeyw4Ichgiuv_3Ig4FRU4Mfs3_-c3Iikfn3R-mOEdgWiQRXUrqYWsRCqkMP5Yr0XoP52-6buAkmQaqPdQlm8BxWKkyA4A4eUMLSJKD6hUGkr4Ods-s0h62NeFP1M0oXxHJxPDv6ghAyQeI2QMsqVYfOjMjChQpcx-RjWpH14uCGi9uu1qhvVUtTdIx482MsIQhzt1RyB_DGUK2nv8y3EhIjBcxLji6n0QWW1T2b3KIit9QVCXPihrkaQTnAkDthihCYNIrcfv0EfpjAX8Z8VruNJcwMgZBMUVK7s9wd2sXwm6D5CKp6NV_bMev4xzh4lC7N_EDlv1xQMOF6BADhu5GBdN_FliAdkWK2AalBbAv-XjdoHSV2GxO7cSs2tvxfsw6vI056pH4cxUfk6k-r3o1VwhoEitCh_ObFtQapukeGY3ChOloTUZKGkyrTC6CXmiTVb0mV6rN2OCHsfaTImieujoVgcMRBarUvyuaVGWzHG2Er_L8b7CY4dCyAY0OHKn3fHNSJaWHiqQ6fLiI4bMqZwL0TB-u35nnwYX7BoBtPv5m5XMV3wq2TRBGfWpD7vV4x5JSvBTdVskD2L7-E3Ghm9BOJ1x6Uu-8UjQEsRoY8bHrCygYy1y07xRVjK6dXlLsVRzJKx2blAO_eUQawwZbGEbuWV1wGALfIdN52meg5Sa5lvws0BngOJvF0sDcnWAEJ1JxIOFjWqMWCwJKbNNngCPH3rFLfV0Qh4KWcfPpzdwF1ymu22IfsQsx5WAC5V-QLoDdQXTmSgxgigZcLA6VVzuYiRNq_LP5wdhdrk1S-JCp3t38xToMi0pO16i5qz6ULHXFdyrIo_s9ge_ZpH-hmT2dnZTVd_fElOGTCDbbvBJcUo5ySyBBasCAKzCEi_q_DEc2PU6fNWPoxiD3ujrEWHrhlkx2LmEcoBabLjFEAAv4_pMzhpfAAE_1xpSeIRBAD6H5ou-pqpUZqfGqguFKthrJgOrjcyBzyAc-DCY19q6k91X5ZFvvbwRlmjTD99DyqIA7r6-JobJx4YojM9REmxMMGnKRViMBYZM2ZnTGcCzGs1pWuTDhvgloP7QitGEQSlFJH6qiIOtffT6neJMGLP9a3WxjzhfvwWfQzaXJAf1FgBR1hfl4OTmIht9Zf2fsVFgMd33Llt92dS3JfkIy1JX6BpIbrfRVIf_-QqXHkTTBHjkNZGl03eYUsfPhCHuz89Byg7nAzYzAvkHx_nRimvBwF_OHpP9Pb_GjOci6tk_dq7IHjpwB8Y5zsZtlh4tBdxyOaRJ9O6at6C8q-BSYyIJqFx-WUPR4rLde4zzaxStgi0AbguNEc2i0V1f3DaDXlh2aZDCemufyYXjIOHr7fvRKugUFfPAR4gHU5N25EXl_B-YYi2mqnUvSFLSq76i3Hdotmoa_0h2FwZZoe-WBA95J_AKDSiKI63x4ouA7t79J9vSbnmbZML2whM-oK2lBJbs-4yzKO3w5kIFw89ZhbJdqe34hkmoRa0dBarr4iRt_pFjnBghdGCkQP5lQTogSriyboT0HaVHfSDAlYn5T4pIu4FgoWSZF1MoNMgnrampNcTKwLTANYA3krkKUxDmXnScCPy1CA5q7mwKvAu3FI3EDYIR0Kbr-Z4L18f2QXneg4sx_d3iQhyk8X3GXqFGw_Zw5i-fd2RiEp0wctPHHQcWHnUX2Am_Gb5VyzAMvlr2wCr-l37S0vUVUzss3IdUH-KmuGzt3InyaWfPX_KmD2PgUlYr9c7izz6tX0nSEi_UFJcF-S-w4j95ptTvrirXTt202-SD5EgBA6l27OMeKTJUWXHstpXT-eujfLET2tyZWc5g9FSMwPg0RS0g8z7dwiOAqVNOmS1CMKs4D5bGsNlKPwpKXGmUPR0IYBobHiHFU47qLHB8RZAN8ozEzmk2KtoWg8_EPpMrrw9yEOhMUlaFB-6JITAwGvmynXujj4dq-5SgiB0z36MPd2Lbn0C_SIZmiMFUiEGzZuebxm4dOlo7YLLX1AZxRXHkvNB1duPpLXtcA0DMIb7nAIKc-oPqw0Hs8ERPehaBMjaMkkn0vFsicTtYw9ZbOhvhAkjJTE4LNxlsQA_WxVnRXVoqO3dzoZHF8iaC3TzldDtP3atcEUOyv0B66f-YycHCpvrJa_bWS3VicpRxjIbQBENntWoOJcHoei-Bbhl7JlEN4leqVlpEAI4TEz45fIsHBcRTpOfmpZC9Ju1ZsduYPpT0BRrXZFJ20-VX0p5CA6riHw-dDeRgiw-NjSuxoohNokTNk_Ua6jKSHjhC3D44uuV7K3s1u7GWEu8Mo4xEPc2_I4CWgWBZM-i8q3v-3zKiq7mSaTdwr-OP5GJXlZPbHwDwibtwClw-1glhylONQ7kkOx2hnDAlDvUNsg8EhdrkOZ34FvI4YyzfvHMkkEPY1a8hg5KQPapxobCpdE2GfS1-_QgmuPw8VmhuLuj1tlcNcnE2JRjpMlCRsOlidkZJ&bs=1",
    "fb_placement_id": "1250440894970236_1965415003472818",
    "ima": {
      "enable": false
    }
  },
  "owner": {
    "id": "x2ir3vq",
    "screenname": "Net Prime",
    "url": "https://www.dailymotion.com/netprime",
    "username": "netprime",
    "avatars": {
      "60": "https://s1.dmcdn.net/d/5000001P-0Ie2/60x60"
    },
    "type": "vpartner",
    "watermark_image_url": "https://s2.dmcdn.net/w/95UPc1YdyDid2s4qT",
    "watermark_link_url": null
  }
}
```

Reply:
I think this error catched by `dailymotion.py` instead by `netverse`

Reply:
```suggestion
```
So this should be removed, no?

Reply:
Yeah, i think the code is redundant as the geo-block will be handled by dailymotion

Reply:
Fixed